flask_static_digest/__init__.py

```diff
@@ -26,28 +26,29 @@ def init_app(self, app):
         app.config.setdefault("FLASK_STATIC_DIGEST_HOST_URL", None)
 
         self.host_url = app.config.get("FLASK_STATIC_DIGEST_HOST_URL")
-        self.static_url_path = app.static_url_path
 
-        self.manifest_path = os.path.join(app.static_folder,
-                                          "cache_manifest.json")
-        self.has_manifest = os.path.exists(self.manifest_path)
+        self.manifests = {}
 
-        self.manifest = {}
+        def ingestScaffold(endpoint, scaffold):
+            if not scaffold.has_static_folder:
+                return
 
-        if self.has_manifest:
-            self.manifest = self._load_manifest(app)
+            manifest_path = os.path.join(scaffold.static_folder,
+                                         "cache_manifest.json")
+            has_manifest = os.path.exists(manifest_path)
 
-        app.add_template_global(self.static_url_for)
+            if has_manifest:
+                manifest = self._load_manifest(scaffold, manifest_path)
+                self.manifests[endpoint] = manifest
 
-    def _load_manifest(self, app):
-        with app.open_resource(self.manifest_path, "r") as f:
-            manifest_dict = json.load(f)
+        ingestScaffold('static', app)
```

Comment:
This project is using double quotes instead of single. Can you use double quotes everywhere?

Reply:
[d06ab59 favoring double quotes and removing html comments](https://github.com/nickjj/flask-static-digest/pull/33/commits/d06ab598dfca84d46029872fa82baf6f77884cfa)